.github/actions/e2e/create-cluster/action.yaml

```diff
@@ -115,7 +115,10 @@ runs:
         withOIDC: true
       addons:
       - name: vpc-cni
+        version: 1.15.0
```

Comment:
We're going to have to keep up with this and update it, which doesn't seem sustainable. Can we just use the default here?